src/main/jte/bookings.jte

```diff
@@ -45,6 +46,11 @@
                                 <td>${booking.getEndDate().toString()}</td>
                                 <td>${booking.getShareRoomType()}</td>
                                 <td>${booking.getPriority()}</td>
+                                <td>
+
+                                    <a href="/bookings/delete/${booking.getId()}" class="btn btn-ghost">
+                                        <?xml version="1.0" encoding="UTF-8"?><svg width="24px" height="24px" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" color="#000000" stroke-width="1.5"><path d="M20 9L18.005 20.3463C17.8369 21.3026 17.0062 22 16.0353 22H7.96474C6.99379 22 6.1631 21.3026 5.99496 20.3463L4 9" fill="#000000"></path><path d="M20 9L18.005 20.3463C17.8369 21.3026 17.0062 22 16.0353 22H7.96474C6.99379 22 6.1631 21.3026 5.99496 20.3463L4 9H20Z" stroke="#000000" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"></path><path d="M21 6H15.375M3 6H8.625M8.625 6V4C8.625 2.89543 9.52043 2 10.625 2H13.375C14.4796 2 15.375 2.89543 15.375 4V6M8.625 6H15.375" stroke="#000000" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"></path></svg></a>
```

Comment:
This image should be in CSS.
See the DaisyUI source for the admin login dropdown icon for an example.
In fact, we may even have an appropriate icon in CSS already